lib/plugins/features/movies/controller.js

```diff
@@ -8,3 +8,16 @@ exports.create = (payload) => {
     return new Movie({ id: movie.id }).fetch();
   });
 };
+
+exports.list = (filter) => {
+  return new Movie().query((qb) => {
+    if (filter.release_year) {
+      qb.where('release_year', filter.release_year);
+    }
+
+    if (filter.title) {
+      qb.whereRaw('lower(title) = ?', filter.title);
```

Comment:
minor thing: often we uppercase psql commands like `LOWER(title)` to distinguish them.